Jenkinsfile

```diff
@@ -81,7 +110,7 @@ node {
     }
     defraUtils.setGithubStatusSuccess()
   } catch(e) {
-    defraUtils.setGithubStatusFailure(e.message)
+      defraUtils.setGithubStatusFailure(e.message)
```

Comment:
Bit of a nitpick,  but extra whitespace has crept in

Reply:
Fixed